### ISSUE TITLE
Calculations hang up in SAD guess

### ISSUE THREAD

**Issue by Madu86** [merzlab/QUICK#113]
The following calculations hang up in SAD guess. This should be fixed. 

```
HF BASIS=6-31G** cutoff=1e-9 denserms=1e-8  zmake GRADIENT DIPOLE CHARGE=0

K         -5.21768       -0.11753       -0.13722
I         -2.87768       -0.11753       -0.13722
```

```
HF BASIS=6-31G** cutoff=1e-9 denserms=1e-8  zmake GRADIENT DIPOLE CHARGE=0

Ca        -2.81913        0.11414       -1.35525
Cl        -1.71913        0.11414       -1.35525
Cl        -3.91913        0.11414       -1.35525
```

**Comment by Madu86** [comment]
```

 @ Read Job And Atom

  KEYWORD=HF BASIS=6-31G** CUTOFF=1E-9 DENSERMS=1E-8  ZMAKE GRADIENT DIPOLE CHARGE=0                                                                                                                              
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.000E+00
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
      GRADIENT CUTOFF  =  0.000E+00
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.000E+00
 BASIS SET = 6-31G**,  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/6-31GSS.BAS

 @ Finish reading job


 @ Begin SAD initial guess

```

**Comment by Altaimorin** [comment]
Find it. In your input `denserms=1e-8` can not be correctly parsed by `rdnml` and renders `quick_method%pmaxrms` to be 0. You can try `denserms=1.0e-8` instead so that the `uscf` for `getmolsad` can converge in about ten iterations for Potassium. By the way Iodine doesn't exist in the 6-31G**.

**Comment by Madu86** [comment]
@Altaimorin I am aware of this and there is another PR open. We have to save the guess for important S block elements. For example, see the following MgCl2 HF/6-31G calculation. The guess takes >90% of time which is totally stupid. 

```
------------- TIMING ---------------
| INITIAL GUESS TIME  =   433.190231000( 96.31%)
| TOTAL SCF TIME      =    16.569304000(  3.68%)
|       TOTAL OP TIME      =    16.551735000(  3.68%)
|             TOTAL 1e TIME      =     0.016103000(  0.00%)
|             TOTAL 2e TIME      =    16.519298000(  3.67%)
|       TOTAL DII TIME      =     0.017246000(  0.00%)
|             TOTAL DIAG TIME    =     0.005770000(  0.00%)
| TOTAL TIME          =   449.782826000
------------------------------------ 
```

**Comment by Madu86** [comment]
> Find it. In your input denserms=1e-8 can not be correctly parsed by rdnml and renders quick_method%pmaxrms to be 0.

@Altaimorin Can you please fix this? We cannot expect users to put decimal numbers all the time. God knows what they will do!

**Comment by agoetz** [comment]
Agreed with Madu. Also, afaik the decimal point is not a requirement in scientific notation, also not in Fortran. So if a user puts 1e-8 and the program crashes, the user is not to blame.

**Comment by agoetz** [comment]
This has been fixed in PR #125 so I will close this issue.
